failsafe/circuit_breaker.py

```diff
@@ -40,14 +40,14 @@ def allows_execution(self):
 
     def record_success(self):
         """
-        Stores an execution success in the state of the CircuitBreaker
+        Function called when the execution succeeded
```

Comment:
> Records an execution success - should be called when operation protected by circuit breaker succeeded. This will reset counter of consecutive failures. Does nothing if circuit breaker is open.

What about something like this?